components/docusign/common.mjs

```diff
@@ -118,6 +118,51 @@ export default {
         config,
       });
     },
+    async getTemplate(baseUri, templateId) {
+      const config = {
+        method: "GET",
+        url: `${baseUri}templates/${templateId}`,
+      };
+      return this._makeRequest({
+        config,
+      });
+    },
+    async listTemplateCustomFields(baseUri, templateId) {
+      const config = {
+        method: "GET",
+        url: `${baseUri}templates/${templateId}/custom_fields`,
+      };
+      return this._makeRequest({
+        config,
+      });
+    },
+    async listTemplateTabs(baseUri, templateId) {
+      const tabs = [];
+      const template = await this.getTemplate(baseUri, templateId);
+      for (const { documentId } of template.documents) {
+        const config = {
+          method: "GET",
+          url: `${baseUri}templates/${templateId}/documents/${documentId}/tabs`,
+        };
+        const response = await this._makeRequest({
+          config,
+        });
+        tabs.push(...Object.entries(response)
+          .filter((tab) => tab[0] !== "signHereTabs")
+          .map((tab) => tab[1])
+          .flat());
+      }
+      return tabs;
+    },
+    async listDocumentFields(baseUri, templateId, documentId) {
+      const config = {
+        method: "GET",
+        url: `${baseUri}templates/${templateId}/documents/${documentId}/fields`,
+      };
+      return this._makeRequest({
+        config,
+      });
+    },
```

Comment:
I don't see where the method `listDocumentFields` is being used.

Reply:
It isn't, I've left it there in case it's needed in the future